docs/en/advanced_guides/datasets/nuscenes.md

```diff
@@ -184,6 +185,40 @@ It follows the general pipeline of 2D detection while differs in some details:
 - Some data augmentation techniques need to be adjusted, such as `RandomFlip3D`.
   Currently we do not support more augmentation methods, because how to transfer and apply other techniques is still under explored.
 
+2. BEV-based
+   BEV, Bird's-Eye-View, is another popular 3D detection paradigm. It directly takes multi-view images to perform 3D detection, for nuScenes, they are `CAM_FRONT`, `CAM_FRONT_LEFT`, `CAM_FRONT_RIGHT`, `CAM_BACK`, `CAM_BACK_LEFT` and `CAM_BACK_RIGHT`. A basic training pipeline of bev-based 3D detection on nuScenes is as below.
+
+```python
+    train_pipeline = [
+        dict(type='LoadMultiViewImageFromFiles',
+             to_float32=True, num_views=6),
+        dict(type='LoadAnnotations3D', ),
+        dict(type='Pack3DDetInputs',
+             keys=['img', 'gt_bboxes_3d', 'gt_labels_3d', ]),
+    ]
+```
+
+To load multiple view of images, a little modification should be made to the dataset.
+
+```python
+data_prefix = dict(
+    pts='samples/LIDAR_TOP',
+    CAM_FRONT='samples/CAM_FRONT',
+    CAM_FRONT_LEFT='samples/CAM_FRONT_LEFT',
+    CAM_FRONT_RIGHT='samples/CAM_FRONT_RIGHT',
+    CAM_BACK='samples/CAM_BACK',
+    CAM_BACK_RIGHT='samples/CAM_BACK_RIGHT',
+    CAM_BACK_LEFT='samples/CAM_BACK_LEFT',
+    sweeps='sweeps/LIDAR_TOP')
+
+dataset = NuScenesDataset(data_root="./data/nuScenes",
```

Comment:
We suggest to use config style, such as `train_dataloader=dict(dataset=dict(...))`

Reply:
You may refer to this [config](https://github.com/open-mmlab/mmdetection3d/blob/main/projects/DETR3D/configs/detr3d_r101_gridmask.py) for reference. Also, we will consider to add a basic dataset config for BEV-based detection on nus.